wcivf/apps/elections/models.py

```diff
@@ -499,4 +499,4 @@ def explainer_url(self):
         elif self.slug == "sv":
             return reverse("sv_voting_system_view")
         else:
-            return ""
+            raise ValueError
```

Comment:
Small point, but it's a good idea to give a message when raising - this ends up getting sent through to sentry / emails / tests / etc and helps debugging.

```raise ValueError(f"{self.slug} has no URL defined")```

Reply:
Thanks! I was wondering about this.